cypress/integration/clue/full/student_tests/canvas_test_spec.js

```diff
@@ -335,15 +335,15 @@ context('Test Canvas', function(){
                     clueCanvas.getShareButton().should('have.length',1)
                 });
                 describe('header actions in 2up view',function(){
-                    it('verify new workspace', function(){
-                        let title = 'New in 2up'
-                        canvas.createNewProblemDocument(title);
-                        clueCanvas.getRightSideWorkspaceTitle().should('contain',title)
-                    });
                     it('verify copy of workspace',function(){
                         let title = 'copy of 1.2'
                         canvas.copyDocument(title);
-                        clueCanvas.getRightSideWorkspaceTitle().should('contain',title)
+                        clueCanvas.getLeftSidePersonalDocTitle().should('contain',title)
```

Comment:
This is really minor but seeing that there is currently a mix use of spacing or lack of, I think it would be preferable to maintain the standard of a space here as the rest of the codebase has that as a linting requirement.
```suggest
                        clueCanvas.getLeftSidePersonalDocTitle().should('contain', title)
```